### ISSUE TITLE
How to get training dataset

### ISSUE THREAD

**Issue by pushkalkatara** [PKU-EPIC/PartManip#1]
Hi, Thanks for the amazing work!

I was trying to run the training scripts but I get this error while running:
```
FileNotFoundError: [Errno 2] No such file or directory: 'assets/drawer/train'
```

How do I setup the training assets?

**Comment by geng-haoran** [comment]
Thank you for your interest in our work! We have already released our assets for PartManip [here](https://forms.gle/DqdPvLE6pNWZf2XR8).
If you have further questions, feel free to contact us.

**Comment by pushkalkatara** [comment]
Hi @geng-haoran , Thanks for sharing the link to the assets.

I think the franka panda arm description is missing from the asset dataset. When I run the training script I get the following error:
```
Creating envs:    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:--*** Failed to load 'assets/franka_description/robots/franka_panda_slider_new.urdf'[Error] [carb.gym.plugin] Failed to parse URDF file 'franka_description/robots/franka_panda_slider_new.urdf'
[Error] [carb.gym.plugin] Failed to import URDF file 'franka_description/robots/franka_panda_slider_new.urdf'

  File "/home/pkatara/PartManip/gym/envs/utils/load_env.py", line 455, in _load_franka
    franka_dof_state["pos"] = (state[:-1, 0])
ValueError: could not broadcast input array from shape (11,) into shape (0,)
```

**Comment by geng-haoran** [comment]
Hi, our modified Franka asset has been uploaded. You can check the form again, also [here](https://forms.gle/DqdPvLE6pNWZf2XR8).

**Comment by pushkalkatara** [comment]
Hi @geng-haoran , thanks for sharing the modified asset.

The above error is resolved but I think the log files are missing in `ppo_utils`.  I get this error while running the training script:
```
  File "/home/pkatara/PartManip/gym/algorithms/pregrasp_ppo/imitation.py", line 33, in <module>
    from ..ppo_utils.log_util import log, log_test
ModuleNotFoundError: No module named 'algorithms.ppo_utils.log_util'
```

**Comment by geng-haoran** [comment]
Yes! My mistake! I accidentally ignored it using git. I update the code just now. Please pull it again! Thank you very much to point it out!!!

**Comment by geng-haoran** [comment]
I test the training script again just now and it runs well! Good luck and contact me if you meet any other questions!

**Comment by pushkalkatara** [comment]
Thanks @geng-haoran for the prompt response.

I tried running the training script without `headless` mode.

For some reason, the base of the Franka Panda arm is not [static](https://drive.google.com/file/d/1nEpLBql4NcyQx0MSO_KjxXWFhc0S--LR/view?usp=sharing) on the table. Is this expected?

**Comment by geng-haoran** [comment]
Yes, we have different versions of Franka, check it in the assets. For OpeningDoor, it's a bit challenging if the base can not move. We also train a model that the base can not move, and use this model in real-world experiments. The performance drops a bit.

**Comment by pushkalkatara** [comment]
Thanks @geng-haoran for the details! I'll try running with the other versions Franka descriptions.

**Comment by pushkalkatara** [comment]
I had another follow-up question:
Is it possible to generate training data for other GAPartNet object categories?

**Comment by geng-haoran** [comment]
Yes! You can download the GAPartNet dataset [here](https://forms.gle/3qzv8z5vP2BT5ARN7) and use the annotated bbox and identify which part to manipulate. You can check the assets format and implement it the same way.

**Comment by pushkalkatara** [comment]
Thanks! This was really helpful.

**Comment by pushkalkatara** [comment]
Hi @geng-haoran ,

I tried training for the `drawer` task referring to the script [here](https://github.com/PKU-EPIC/PartManip/blob/main/scripts/train.sh#L2), but for some reason unable to get the same success rate in 60k steps.
I trained till 3M but convergence seem slow. [here](https://wandb.ai/pushkalkatara/PartManip-gym/runs/1gq35jk6?workspace=user-pushkalkatara) are the wandb logs for my run. 

what might go wrong while training? is it related to the number of envs spawned while training? in my case, 30env are spawned with 6 assets and 5 replications.
